bears/vcs/git/GitMergeInspectBear.py

```diff
@@ -0,0 +1,38 @@
+from bears.vcs.VCSCommitBear import COMMIT_TYPE
+from bears.vcs.git.GitCommitBear import GitCommitBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class GitMergeInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitBear}
+
+    def run(self, dependency_results,
+            allow_git_merge_commit: bool = False,
+            **kwargs):
+        """
+        Inspect the HEAD commit to check if it is a
+        git merge commit and return result accordingly.
+
+        :param allow_git_merge_commit: Whether merge commits are
+                                       allowed or not.
+        """
+        if allow_git_merge_commit:
+            return
+
+        gitrebase_tutorial_link = (
```

Comment:
Make this as a constant inside of the `GitMergeInspectBear`, e.g `GIT_REBASE_TUTORIAL_LINK`